parse/parse.go

```diff
@@ -50,6 +50,7 @@ type Message interface {
 	MsgType() string
 	MeterID() uint32
 	MeterType() uint8
+	MeterValue() uint32
```

Comment:
Checking for duplicate messages against consumption value can fail if the consumption value hasn't changed but other fields in the message have, this is especially true for IDM where usage intervals can progress while the consumption value hasn't. We should check against checksum instead which typically covers the entire packet.

This will be an easy change for SCM and IDM but R900 will need to have a new field added.